from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_Dialog_Processing(object):
    def setupUi(self, Dialog_Processing):
        
        Dialog_Processing.setObjectName("Dialog_Processing")
        Dialog_Processing.setFixedSize(330, 180)
        self.progressBar = QtWidgets.QProgressBar(Dialog_Processing)
        self.progressBar.setGeometry(QtCore.QRect(20, 90, 301, 25))
        self.progressBar.setProperty("value", 0)
        self.progressBar.setObjectName("progressBar")
        palette = QtGui.QPalette()
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(85, 170, 255))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.Button, brush)
        brush = QtGui.QBrush(QtGui.QColor(85, 170, 255))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.Window, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(85, 170, 255))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.Button, brush)
        brush = QtGui.QBrush(QtGui.QColor(85, 170, 255))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.Window, brush)
        brush = QtGui.QBrush(QtGui.QColor(42, 85, 127))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(85, 170, 255))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.Button, brush)
        brush = QtGui.QBrush(QtGui.QColor(85, 170, 255))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.Window, brush)
        Dialog_Processing.setPalette(palette)
        Dialog_Processing.setAutoFillBackground(True)
        self.label_OngoingOperation = QtWidgets.QLabel(Dialog_Processing)
        self.label_OngoingOperation.setGeometry(QtCore.QRect(30, 10, 271, 31))
        font = QtGui.QFont()
        font.setFamily("FRIZON")
        font.setPointSize(15)
        font.setUnderline(True)
        self.label_OngoingOperation.setFont(font)
        self.label_OngoingOperation.setFocusPolicy(QtCore.Qt.NoFocus)
        self.label_OngoingOperation.setLayoutDirection(QtCore.Qt.LeftToRight)
        self.label_OngoingOperation.setTextFormat(QtCore.Qt.AutoText)
        self.label_OngoingOperation.setAlignment(QtCore.Qt.AlignCenter)
        self.label_OngoingOperation.setObjectName("label_OngoingOperation")
        self.label_Processing = QtWidgets.QLabel(Dialog_Processing)
        self.label_Processing.setGeometry(QtCore.QRect(20, 60, 261, 21))
        font = QtGui.QFont()
        font.setFamily("FRIZON")
        font.setPointSize(12)
        self.label_Processing.setFont(font)
        self.label_Processing.setFocusPolicy(QtCore.Qt.NoFocus)
        self.label_Processing.setLayoutDirection(QtCore.Qt.LeftToRight)
        self.label_Processing.setTextFormat(QtCore.Qt.AutoText)
        self.label_Processing.setAlignment(QtCore.Qt.AlignLeading|QtCore.Qt.AlignLeft|QtCore.Qt.AlignVCenter)
        self.label_Processing.setObjectName("label_Processing")
        self.pushButton_Cancel = QtWidgets.QPushButton(Dialog_Processing)
        self.pushButton_Cancel.setGeometry(QtCore.QRect(20, 130, 101, 31))
        palette = QtGui.QPalette()
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 70, 70))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.Button, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 70, 70))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.Window, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 70, 70))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.Button, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 70, 70))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.Window, brush)
        brush = QtGui.QBrush(QtGui.QColor(127, 35, 35))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 70, 70))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.Button, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 70, 70))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.Window, brush)
        self.pushButton_Cancel.setPalette(palette)
        font = QtGui.QFont()
        font.setPointSize(10)
        self.pushButton_Cancel.setFont(font)
        self.pushButton_Cancel.setAutoFillBackground(True)
        self.pushButton_Cancel.setObjectName("pushButton_Cancel")
        self.retranslateUi(Dialog_Processing)
        QtCore.QMetaObject.connectSlotsByName(Dialog_Processing)

    def retranslateUi(self, Dialog_Processing):
        _translate = QtCore.QCoreApplication.translate
        Dialog_Processing.setWindowTitle(_translate("Dialog_Processing", "Processing"))
        self.label_OngoingOperation.setText(_translate("Dialog_Processing", "Ongoing operation"))
        self.label_Processing.setText(_translate("Dialog_Processing", "Processing..."))
        self.pushButton_Cancel.setText(_translate("Dialog_Processing", "Cancel"))

        


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    app.setStyle('Fusion')
    Dialog_Processing = QtWidgets.QDialog()
    ui = Ui_Dialog_Processing()
    ui.setupUi(Dialog_Processing)
    Dialog_Processing.show()
    sys.exit(app.exec_())
